.github/workflows/build.yaml

```diff
@@ -37,8 +37,8 @@ jobs:
           tag=${version}-${base}_edge
           yq e -i ".version = \"$tag\"" rockcraft.yaml
           rockcraft pack --verbose
```

Comment:
for future reference: there's a rockcraft action that might be useful: https://github.com/canonical/craft-actions/tree/main#rockcraft-pack